SerialPrograms/Source/PokemonSV/Programs/Sandwiches/PokemonSV_SandwichRoutines.cpp

```diff
@@ -32,7 +32,7 @@ namespace PokemonAutomation{
 namespace NintendoSwitch{
 namespace PokemonSV{
 
-namespace {
+//namespace {

```

Comment:
why are you removing this ? This is a cpp anyway and nothing else from this file seem to have changed

Reply:
anonymous namespace preventing access to the function from the sandwich maker

Reply:
oh nevermind, the implementation part was in a namespace but not the declaration, i didn't knew it was valid !

Reply:
EDIT : it wasn't, i'm just stupid and those were the function I searched in the header, must have been tired xD